### ISSUE TITLE
Chinese, Japanese and Korean are not properly handled

### ISSUE THREAD

**Issue by flyingns** [vilcans/screenplain#74]
I am an Asian. When I use screenplain to convert scripts containing Chinese, Japanese and Korean, the pdf text converted by screenplain is all black squares. Can you solve this problem?

<img width="527" alt="Screen Shot 2022-11-02 at 22 31 24" src="https://user-images.githubusercontent.com/16456728/199519834-e276c777-89e1-4737-aaec-f9c203a148d9.png">
<img width="369" alt="Screen Shot 2022-11-02 at 22 31 51" src="https://user-images.githubusercontent.com/16456728/199519910-f1c5f3a3-0640-433e-b21d-01baef6bb82c.png">

Thank you for everything.

**Comment by vilcans** [comment]
Yes, as far as I can tell this is because the Courier font doesn't have those characters. I'm working on adding support other fonts, which should solve this problem.

**Comment by lmanhes** [comment]
Hi @vilcans, thank you for your work!

Do you have any idea how to solve this problem?
Can i help?

**Comment by vilcans** [comment]
Please check if this is still an issue in the latest version which loads an external font. Also, can you send an example fountain file in these languages to test on?
